src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -258,6 +253,8 @@ public void execute(String newHtmlPath)
 
    public void onDeactivate()
    {
+      commands_.saveProfileAs().setEnabled(false);
```

Comment:
We should be able to handle this via use of our dynamic commands (commands which are only enabled by file type / active editing target). Here is where dynamic commands are defined (you need to add saveProfileAs to this):

https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java#L280

And here is where you indicate that the profiler file type includes support for this command:

https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/common/filetypes/ProfilerType.java#L50-L56